### PR TITLE
[MU4] New score wizard stabilization

### PR DESCRIPTION
**Pull request by RomanPudashkin**

### CODE REVIEW

File: src/userscores/view/additionalinfomodel.h [hunk 1]
```diff
@@ -163,6 +165,30 @@ public slots:
         }
     };
 
+    struct Tempo {
+        int value = 0;
+        ui::MusicalSymbolCodes::Code noteIcon;
```
Comment (Eism) [inline src/userscores/view/additionalinfomodel.h:126]: please add default value

File: src/userscores/qml/MuseScore/UserScores/internal/KeySignature.qml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import QtQuick 2.12
```
Comment (Eism) [inline src/userscores/qml/MuseScore/UserScores/internal/KeySignature.qml:1]: 2.15, please

**Review by Eism — APPROVED** [review]

File: src/userscores/qml/MuseScore/UserScores/internal/KeySignature.qml (hunk 2 above)
Comment (RomanPudashkin) [reply src/userscores/qml/MuseScore/UserScores/internal/KeySignature.qml]: fixed

File: src/userscores/view/additionalinfomodel.h (hunk 1 above)
Comment (RomanPudashkin) [reply src/userscores/view/additionalinfomodel.h]: fixed

**Pull request merged**